tests/job_timeout_test.py

```diff
@@ -53,3 +54,24 @@ def test_srw(fc):
         if o:
             pkdlog('found "mpiexec" after cancel in ps={}', '\n'.join(o))
             raise AssertionError('cancel failed')
+
+
+def test_myapp_analysis(fc):
+    from pykern import pkunit
+    from pykern.pkdebug import pkdp
+
+    d = fc.sr_sim_data()
+    r = fc.sr_run_sim(d, 'heightWeightReport', expect_completed=True)
+    r = fc.sr_get(
+        'downloadDataFile',
+        PKDict(
+            simulation_type=d.simulationType,
+            simulation_id=d.models.simulation.simulationId,
+            model='heightWeightReport',
+            frame='-1',
+            suffix='sr_long_analysis',
+        ),
+    )
+    # TODO(e-carlin): This should be something more helpful than a 404
```

Comment:
What's the best option here? 404 seems "fine" but not the most helpful.

Reply:
I think this is a very rare case so 404 is fine.